components/brave_shields/common/brave_shields_panel.mojom

```diff
@@ -26,5 +26,16 @@ interface PanelHandler {
 };
 
 // WebUI-side handler for requests from the browser.
-interface Page {
+interface UIHandler {
+  OnSiteDataChanged(CurrentSite current_site);
+};
+
+interface DataHandler {
+  RegisterUIHandler(pending_remote<UIHandler> ui_handler);
+  GetCurrentSiteData() => (CurrentSite current_site);
+};
+
+struct CurrentSite {
```

Comment:
`SiteBlockInfo`? Or something that doesn't make me think this is a struct that describes a `Site`?

Reply:
use SiteBlockInfo